src/java.base/share/classes/java/lang/invoke/VarForm.java

```diff
@@ -109,9 +109,12 @@ final MethodType getMethodType(int type) {
 
     @ForceInline
     final MemberName getMemberName(int mode) {
-        MemberName mn = getMemberNameOrNull(mode);
+        MemberName mn = memberName_table[mode];
```

Comment:
It makes sense to leave a comment here why inlining `getMemberNameOrNull()` makes sense.  Otherwise, looks good.

Reply:
Added comment.